relation_engine_server/utils/arango_client.py

```diff
@@ -100,27 +100,35 @@ def create_collection(name, config):
         _create_indexes(name, config)
 
 
-def _create_indexes(coll_name, config):
-    """Create indexes for a collection"""
-    url = _CONF["api_url"] + "/index"
-    # Fetch existing indexes
-    auth = (_CONF["db_user"], _CONF["db_pass"])
-    resp = requests.get(url, params={"collection": coll_name}, auth=auth)
+def _get_indexes(coll_name):
+    """Fetch existing indexes for a collection"""
+    resp = requests.get(
+        url=_CONF["api_url"] + "/index",
+        params={"collection": coll_name},
+        auth=(_CONF["db_user"], _CONF["db_pass"]),
```

Comment:
if this is used numerous times in this file, it might be worth turning it into a variable

Reply:
This was originally because I was thinking of adding a check on ArangoDB server startup to check that all the indexes in the specs are existing as configured, since the indexes/analyzers/views are, in live deploy, actually manually added by an admin. I actually have no idea how I would actually do this in tests though because the indexes/analyzers/views are added in `setUpClass` so I might abandon this effort unless someone objects or can help.